compiler/q-implant/README.md

```diff
@@ -2,13 +2,54 @@
 
 _q-implant_ writes quantization parameters and weights (given as .json and .npy files) to a circle model.
 
+## Format of input files (`.json` and `.npy`)
+The main input file is a `.json` file, which is a dictionary whose key is a tensor name and value is quantization parameters and values (if exists).
```

Comment:
plz split lines to be within 100 or 80 cols.

Reply:
> plz split lines to be within 100 or 80 cols.

I split the line.
But the second sentence(line 9) is 102 letters, but I think it would be weird to divide it more.
So I'll leave it as it is.